Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py

```diff
@@ -891,13 +891,25 @@ def get_indicators(**kwargs):
         Returns filtered indicators by parameters from ThreatStream.
         By default the limit of indicators result is set to 20.
     """
+    limit = int(kwargs.get('limit', 20))
     if 'query' in kwargs:
-        params = build_params(q=kwargs['query'], limit=kwargs.get('limit', 20))
+        params = build_params(q=kwargs['query'], limit=limit)
     else:
         params = build_params(**kwargs)
 
     iocs_list = http_request("GET", "v2/intelligence/", params=params).get('objects', None)
 
+    if limit > 1000:
+        limit -= 1000
+        offset = 1000
```

Comment:
you should handle numbers like 20000 as well

Reply:
done